04_instrumented.md

```diff
@@ -383,16 +383,24 @@ and they all use a distinct algorithm:
 
 - [`std::partial_sort`](https://en.cppreference.com/w/cpp/algorithm/partial_sort)
 
-    For example, you give it 100 elements and you sort from 1st to the
-    10th to the last. But, for the last 90 elements there is no guarantee.
+    What's the interface?
+    It takes three iterators `first`, `middle`, and `last`.
+    What does it do?
+    It orders the elements so that `first` to `middle`
+    contains the smallest elements from the entire range.
+    Then it sorts those first elements.
```

Comment:
This seems to imply that there are two distinct phases to the algorithm. Perhaps just shorten it to this?
```suggestion
    It orders the elements so that `first` to `middle`
    contains the smallest elements from the entire range, in sorted order.
```